bumble/utils.py

```diff
@@ -64,6 +65,48 @@ def deregister(self, emitter):
     return cls
 
 
+_F = TypeVar('_F', bound=Callable)
```

Comment:
suggest: maybe call this `_Handler` to make it clear what it is used for.

Reply:
Done